public/javascript/ep-card-filter.js

```diff
@@ -194,9 +199,45 @@ $(document).ready(function(){
       window.cards.saveState();
     });
 
-    $('[data-section-toggle]').on('click', function(){
-      var section = $(this).attr('data-section-toggle');
+    $('.js-show-facet').on('change', function(){
```

Comment:
I can only see two others in this file, but I guess that does make it the majority 😉  

I agree, consistency is best for this pull request, we can change to the shorthand methods in a future PR 👍